shared/src/helpers/sanitize/sanitize-html/sanitize-html.helper.ts

```diff
@@ -0,0 +1,7 @@
+import { sanitize } from 'isomorphic-dompurify';
+
+const sanitizeHTML = (html: string): string => {
+  return sanitize(html, {});
```

Comment:
If you're not passing any additional config options
```suggestion
  return sanitize(html);
```

Reply:
but then an error occurs
 
![image](https://user-images.githubusercontent.com/56650388/186145939-53dbbc97-40a5-4fcc-82ea-4c474fefb6bb.png)

Reply:
Okay 😕